packages/shared/src/components/Greeting.tsx

```diff
@@ -75,7 +75,7 @@ export default function Greeting({
 
   useEffect(() => {
     (async () => {
-      const media = window.matchMedia(tablet.replace('@media ', ''));
+      const media = window.matchMedia(laptopL.replace('@media ', ''));
```

Comment:
It would lead to more dependencies in useEffect so in this case I would also leave it as is. Also to minimize code changes. The above component could use the refactor for sure 👍